ONSdigital/eq-author
# 39/Navigation

### What is the context of this PR?

https://trello.com/c/RjGigMDa/39-navigation-5

All AC for this story are already covered by other stories. This PR adds page transitions when navigating as well as some visual tweaks as agreed with Dan.

### How to review 

1. Navigate to or create a Q with multiple pages/sections
2. Check navigation between pages works and has suitable transitions
